torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -564,11 +564,14 @@ at::Tensor AtenXlaType::avg_pool2d(const at::Tensor& self,
                                    at::IntArrayRef kernel_size,
                                    at::IntArrayRef stride,
                                    at::IntArrayRef padding, bool ceil_mode,
-                                   bool count_include_pad) {
+                                   bool count_include_pad,
+                                   c10::optional<int64_t> divisor_override) {
+  // devisor_override is set not supported yet.
```

Comment:
`devisor_override` -> `divisor_override`

Reply:
I'd just drop this comment as it does not add much one cannot read from the code.
Same in all other instances of this PR.